src/cli/cli.cpp

```diff
@@ -2984,7 +2984,7 @@ otError Interpreter::ProcessNetworkTime(Arg aArgs[])
 
         networkTimeStatus = otNetworkTimeGet(mInstance, &time);
 
-        OutputFormat("Network Time:     %luus", time);
+	OutputFormat("Network Time:     %luus", time);
```

Comment:
Code formatting is bad. I think you can run `./script/make-pretty check` to test your patch.